rootfs/api/models/app.py

```diff
@@ -792,7 +792,10 @@ def _build_env_vars(self, release):
         # mix in default environment information deis may require
         default_env = {
             'DEIS_APP': self.id,
-            'WORKFLOW_RELEASE': 'v{}'.format(release.version)
+            'WORKFLOW_RELEASE': 'v{}'.format(release.version),
+            'WORKFLOW_RELEASE_SUMMARY': release.summary,
+            'WORKFLOW_RELEASE_CREATED_AT': str(release.created.strftime(
+                settings.DEIS_DATETIME_FORMAT))
```

Comment:
Does this need to go through `strftime`? Just curious since we set the format already for DRF so I'd expect it to translate it for us when we do `str()` on it, but maybe not?